bmwqemu.pm

```diff
@@ -126,7 +126,7 @@ sub init {
 
     unless ($vars{CASEDIR}) {
         die "DISTRI undefined\n" . pp(\%vars) . "\n" unless $vars{DISTRI};
-        my @dirs = ("$scriptdir/distri/$vars{DISTRI}");
+        my @dirs = ("$scriptdir/tests/$vars{DISTRI}");
```

Comment:
I don't get this check at all. We don't have `distri` directory AFAIK, but use tests instead. So this test IMO never passed. `$striptdir` is also set to some `/usr/lib/os-autoinst` instead of `/var/lib/openqa[/share]`. I'll remove this part and consider `CASEDIR` to be mandatory.